### ISSUE TITLE
DSU server motion controls only work when window is focused

### ISSUE THREAD

**Issue by ashkeplussen** [Valkirie/HandheldCompanion#679]
On a Windows Steam Deck. While the DSU server seems to be able to connect to emulators at all times, actual motion data doesn't seem to be sent unless the HandheldCompanion window is open and in focus.

**Comment by Valkirie** [comment]
Yes that's the case right now. Steam Deck gyro values are sent as part of its controller inputs and therefore managed by the UI, not the service. We might be able to do something about it one day though.

**Comment by CasperH2O** [comment]
@Valkirie , I can't test this one myself, lacking a SD, but now that the service and UI are combined, is still issue still present?

**Comment by SisterVersio** [comment]
Still an issue
